specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -51,6 +51,555 @@
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcsRouterWorkerSelector"
+          }
+        },
+        "requestedWorkerSelectors": {
+          "description": "Router Job Queued Requested Worker Selector",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcsRouterWorkerSelector"
+          }
+        }
+      }
+    },
+    "AcsRouterJobRecievedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterJobReceived event",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterJobEventBaseProperties"
+        }
+      ],
+      "properties": {
+        "jobStatus": {
+          "description": "Router Job Received Job Status",
+          "type": "string"
+        },
+        "classificationPolicyId": {
+          "description": "Router Job Classification Policy Id",
+          "type": "string"
+        },
+        "priority": {
+          "description": "Router Job Priority",
+          "type": "integer",
+          "format": "int32"
+        },
+        "requestedWorkerSelectors": {
+          "description": "Router Job Received Requested Worker Selectors",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcsRouterWorkerSelector"
+          }
+        },
+        "scheduledTimeUtc": {
+          "description": "Router Job Received Scheduled Time in UTC",
+          "type": "string",
+          "format": "date-time"
+        },
+        "unavailableForMatching": {
+          "description": "Unavailable For Matching for Router Job Recieved",
+          "type": "boolean"
+        }
+      }
+    },
+    "AcsRouterJobSchedulingFailedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterJobSchedulingFailed event",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterJobEventBaseProperties"
+        }
+      ],
+      "properties": {
+        "priority": {
+          "description": "Router Job Priority",
+          "type": "integer",
+          "format": "int32"
+        },
+        "attachedWorkerSelectorsExpired": {
+          "description": "Router Job Scheduling Failed Attached Worker Selector Expired",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcsRouterWorkerSelector"
+          }
+        },
+        "requestedWorkerSelectorsExpired": {
+          "description": "Router Job Scheduling Failed Requested Worker Selector Expired",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcsRouterWorkerSelector"
+          }
+        },
+        "scheduledTimeUtc": {
+          "description": "Router Job Scheduling Failed Scheduled Time in UTC",
+          "type": "string",
+          "format": "date-time"
+        },
+        "failureReason": {
```

Comment:
Should this be an enum that uses x-ms-enum with "modelAsString"= true?

Reply:
no not an enum

Reply:
In all the examples, it has a single value. This should probably be an enum or we need to provide better examples.